### DISCUSSION TITLE
Having trouble starting SwimClubMeet

### DISCUSSION THREAD

**Discussion by MCG2024** [Artanemus/SCM_SwimClubMeet-R#7] [Q&A] [answered]
I'm following the manual to set up the program. I've created the server, ran BuildMeAClub and created a club, but when I log into SwimClubMeet I get the error message "qryEvent: Field 'SCHED' not found"

Not sure what I'm doing wrong... any help is greatly appreciated! I'm not the most computer savvy and trying to use the program to run our local swim meets. Thank you!

**Comment by Artanemus** [comment, marked as answer]
As the DATABASE version changed from 1.1.5.1 to 1.1.5.2, a new field called ScheduleDT was added to the table called dbo.Event. I've check my build that creates this table (for version 1.1.5.2) and it is indeed part of the SQL script. This is what I suggest you do.

- Have you entered lots of data in the database? If you have - then stop - drop me another comment?
- Delete the database from the server. (Select the database in MSSMS, right click, select delete, tick close 'existing connections', press OK )
- Go to the Artanemus/SCM_BuildMeAClub repository. 
- DON'T click on the  SCM_BuildMeAClub_x64_v1.5.2.0 LATEST instead click on the text line that says 'Releases 9". It'll take you to a FULL LIST of RELEASES AND PRE-RELEASES. 
- Select the pre-release v2.0.1.3 - click ok
- Make sure to uninstall any previous version of BuildMeAClub before installing v2.0.1.3.

This pre-release is a 'Universal' BuildMeAClub. It has all the versions of the DATABASE. Including the one you want - 1.1.5.2

[Screenshot of v2.0.1.3 with select database version dialogue box](https://github.com/Artanemus/SCM_BuildMeAClub-R/blob/main/ASSETS/Screenshot%20MainForm_SelectDB.JPG)

Here is a quick link to v2.0.1.3
https://github.com/Artanemus/SCM_BuildMeAClub-R/releases/tag/v2.0.1.3

If this work's for you or it doesn't - please drop another comment .  :-) 

I'll will do a RELEASE version of v2.0.1.3 soon. Currently I'm testing  'SCM_UpdateDatabase' - this will take existing users on any DATABASE version and allow them to upgrade to v1.1.5.3 (this has team events - relays and other goodies) 

Cheers
Artanemus

And thanks for given the app a go - you won't be disappointed.

**Reply by MCG2024** [reply]
This worked for me! I had not entered any data in the database, but I deleted it anyway just in case. The previous version worked seamlessly. I'm able to add members now and excited to try the app. Thank you so much!

**Reply by MCG2024** [reply]
Application is working great so far! Is there a way to create a session and have different clubs add members to that session? I'm able to add members from my own club but what if I wanted to run a meet with multiple clubs? Thanks in advance!

**Reply by Artanemus** [reply]
I'm glad to hear you work it all out. I'm sorry that installing SCM can be challenging but it's worth it.  

The answer to the multi-club question - is no, BUT. (I don't like to announce stuff I haven't finished - so lets keep this a secret)

- In the SCM database there is a table called dbo.SwimClub. It can hold many clubs. Currently, SCM only uses the club with the ID of 1. Currently the members table is linked via a one-to-one relationship. In version 1.1.5.3 this relationship is changed to one-to-many. Which means a swimmer can be a member of more than one club. Future SCM apps will allow you to run more than one club. This means  if you have short course over winter and long course over summer, you will build two clubs. And the 'pool' of members can be shared over those clubs.
- There will be an application called SCM_CARNIVAL. This application look identical to SwimClubMeet, with this feature, it runs a session with many clubs competing together. Clubs export from SCM (via USB or on the cloud etc.) a snap-shot of club members names and PBs. Then those members-profiles are loaded into SCM_CARNIVAL. So we can all have a club-vs-clubs carnival. AND for those driving the CARNIVAL application - it's easy - it looks the same as SCM, nothing new to learn!

When will all this happen? When I get time. (I am one of those sort of people that does finish stuff they start.)
Cheers
Artanemus
